Source/Charts/Charts/BarChartView.swift

```diff
@@ -180,4 +180,8 @@ open class BarChartView: BarLineChartViewBase, BarChartDataProvider
     
     /// - returns: `true` if drawing shadows (maxvalue) for each bar is enabled, `false` ifnot
     open var isDrawBarShadowEnabled: Bool { return drawBarShadowEnabled }
+
+    open var barRoundingCorners: UIRectCorner = []
```

Comment:
This will cause a problem for Mac apps, should avoid using `UIKit` in these circumstances